index.js

```diff
@@ -28,3 +28,110 @@ links.forEach((link) => {
+  },
+  {
+    title: 'multi-post stories',
+    image: 'images/Snapshootportfolio(3).png',
+    paragraph: 'A daily selection of privately personalised reads: no accounts or sign-ups required',
+  },
+];
+
+const projectContainer = document.querySelector('.project-container');
+
+const projectHtml = projects.map((project) => (
+  `
+  <div class="project">
+  <div class="my-img">
+      <img class="mobile-image" src=${project.image} alt="picture identity of a lady">
+  </div>
+  <div class="my-text">
+      <h2>${project.title}</h2>
+      <ol>
+          <li>CANOPY</li>
+          <li><img src="images/Counter.png" alt="counter"></li>
+          <li>Back End Dev</li>
+          <li><img src="images/Counter(2).png" alt="counter"></li>
+          <li>2015</li>
+      </ol>
+      <p>${project.paragraph} </p><br>
+      <ul class="show">
+          <li>html</li>
+          <li>css</li>
+          <li class="ok">Ruby on rails</li>
+          <li>javascrpt</li>
+      </ul>
+      <button class="active button" type="button"> See Project</button>
+  </div>
+</div>
+  `
+));
+
+projectContainer.innerHTML = projectHtml;
+
+const projectButtons = document.querySelectorAll('.button');
+
+const popupContainer = document.querySelector('.popup');
+
+projectButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    popupContainer.innerHTML = `
+    <div class="project1">
+    <img class="popupx" src="images/Iconx-popup.png" alt="an x">
+    <h2>Tonic</h2>
+    <ul class="list1">
+        <li>CANOPY</li>
+        <li><img src="images/Counter.png" alt="counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="images/Counter(2).png" alt="counter"></li>
+        <li>2015</li>
+    </ul>
+    <div class="my-img1">
+        <img class="mobile-image1" src="images/SnapshootPortfolio(1).png" alt="picture identity of a lady">
+    </div>
+    <div class="my-img2">
+    <img class="mobile-image2" src="images/Snapshoot-desktop4.png" alt="nature picture">
+    </div>
+    <p>
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+        industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+        scrambled it to make a type specimen book.It has survived not only five centuries, but also
+        the leap into electronic typesetting, remaining essent</p>
+    <ul class="show1">
+        <li>html</li>
+        <li>css</li>
+        <li class="ok1">Ruby on rails</li>
+        <li>javascrpt</li>
+    </ul>
+    <div class="house">
+        <div class="button1a">
+            <button type="button" class="button1">See live <img src="images/Icon-popup.png" alt="a clock"></button>
+        </div>
+        <div class=" button1a">
+            <button type="button" class="button2">See Source <img src="images/Vector-popup.png" alt="github">
+            </button>
+        </div>
+    </div>
+</div>
```

Comment:
- [ ] Following the project requirements, your modal popup data is populated with the data from the projects array. This means that the `popup` will contain data from its specific card eg: `title`, `image`, `links`, etc. In your case, all your cards have the same popup with the same data. Kindly, change it and ensure that data is populated dynamically because this will help you add your projects in the future. You will just need to add it in one place. This will be cool no 😊 
**Project requirements**

![image](https://user-images.githubusercontent.com/80612925/175435947-2e2120bd-b680-4320-a556-492c8b053976.png)